uint8 comandoPitch
uint8 comandoRoll
uint8 comandoAltura
